ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxApplication.scala

```diff
@@ -93,7 +97,8 @@ object SandboxApplication {
         }
 
       val (ledgerType, ledger) = config.jdbcUrl match {
```

Comment:
does this mean that metrics are always on? what are the consequences? would people ever want them to be off?

Reply:
Metrics are generally super light-weight and not visible to everyday users. I see no reason not have them enabled all the time. I had applications before having more than 1k metrics and here we are talking about not even a handful of them.

Reply:
comment added: https://github.com/digital-asset/daml/pull/646/commits/803f162b2123524a84ab2731dbc4b99db22ed7b8